### PR TITLE
Finnish fixes 9

### PR DESCRIPTION
**Pull request by fergusq**
This update contains improvements on Finnish definitions and notes of military terms and some other improvements.

Translating military terms is especially difficult as the Finnish military ranks do not match the US ranks (that are used as English translations), causing a sort of double error, as first the Klingon ranks are confusingly translated to English, and then those ranks are loosely translated to Finnish.

Related to inflecting embedded words: In many notes I have translated the embedded word and added the Klingon word in parenthesis: for example `This is the 4th rank of {yaS:n}.` -> `Tämä on upseeriston ({yaS:n}) neljänneksi korkein sotilasarvo.` This is because the word `{yaS:n}` should be inflected in Finnish, but inflecting Klingon words using Finnish suffixes does not work well in boQwI' (eg. `{yaS:n}:n` is rendered as `yaS (subs):n` which looks odd, it should be `yaSin`...), so I prefer adding a Finnish word to be inflected.

Related to the word **wey**: the English definition is "company (military unit)" (in Finnish _komppania_), but the English word can also refer to a ship's crew, which KGT confirms to be the correct meaning. Therefore I translated it as _(aluksen) miehistö_ "company (of a ship)" in the Finnish definition. It should be considered to translate it this way in the English definition as well. I'm not sure of the policy on which English definitions are changed. I have taken freedoms in the Finnish definitions and translated them non-literally when the literal translation is only confusing, but maybe you want to preserve the original translations from the books.

### CODE REVIEW

File: mem-16-t.xml [hunk 1]
```diff
@@ -9861,7 +9861,7 @@ Nousu alaspäin ilmaistaan ​​negatiivisella kulmalla (katso {taH:v:2}). [AUT
       <column name="notes_sv">Detta är den tredje rankningen av {yaS:n}. [AUTOTRANSLATED]</column>
       <column name="notes_ru">Это 3-й ранг {yaS:n}. [AUTOTRANSLATED]</column>
       <column name="notes_zh_HK">這是{yaS:n}的第三等級。 [AUTOTRANSLATED]</column>
-      <column name="notes_pt">Este é o terceiro ranking do {yaS:n}. [AUTOTRANSLATED]</column>
+      <column name="notes_pt">Tämä on upseeriston ({yaS:n}) kolmanneksi korkein sotilasarvo.</column>
```
Comment (De7vID) [inline mem-16-t.xml:9864]: This is in the wrong field.

**Comment by De7vID** [comment]
> Related to the word wey: the English definition is "company (military unit)" (in Finnish komppania), but the English word can also refer to a ship's crew, which KGT confirms to be the correct meaning. 

It's not "also". The term "company" (in its sense as a military unit) refers to a ship's officers and crew. That's the standard English definition For example, here's [Merriam-Webster](https://www.merriam-webster.com/dictionary/company):
image

**Comment by fergusq** [comment]
For example Wiktionary shows the other non-crew meaning before the crew-meaning. I understood that companies are military units in the army, not the navy, but I admit I have poor understanding of English military terms.

kuva

**Comment by De7vID** [comment]
Right, there's an army definition and a navy definition. The definition is following in Star Trek's footsteps of treating spacecraft as "ships in space". I don't know what specifically led Dr. Okrand to add "company" to the vocabulary, but in Star Trek II, Khan speaks of the Botany Bay's "ship's company", and many vocabulary terms were added in Klingon because they were used in Star Trek. I think the English definition is fine.

**Pull request merged**